python_modules/dagster/dagster/_core/scheduler/instigation.py

```diff
@@ -377,6 +386,26 @@ def dynamic_partitions_request_results(
     ) -> Sequence[DynamicPartitionsRequestResult]:
         return self.tick_data.dynamic_partitions_request_results
 
+    @property
+    def asset_materialization_count(
+        self,
```

Comment:
If you take out this comma, black can consolidate the function signature to a single line.